pcs/lib/cib/acl.py

```diff
@@ -325,7 +329,9 @@ def get_role_list(acl_section):
     acl_section -- etree node
     """
     output_list = []
-    for role_el in acl_section.findall("./{0}".format(TAG_ROLE)):
+    for role_el in acl_section.xpath(
+        "./*[local-name()=$tag_name]", tag_name=TAG_ROLE
+    ):
```

Comment:
Here it's not necessary to use xpath variable injection as `TAG_ROLE` is internal constant and not an input from user. In my opinion it makes the code just a little bit more complex and harder to read than necessary. This is also the case in other places.

On the other hand, maybe it's safer to do it for everything without thinking about source of the value. So this is not really a disagreement, just an observation.

Reply:
It's safer to pass everything as variables. In this very case, however, it's an over-complication.